src/app/queries/BootcampFactory/fetchBootcampsAddresses.ts

```diff
@@ -0,0 +1,19 @@
+import { contractFactoryAddress } from "@/app/const";
+import { Address, parseAbiItem, PublicClient } from "viem";
+
+export async function fetchBootcampsAddresses(publicClient: PublicClient | undefined): Promise<Address[] | Error> {
+    if (!publicClient) {
+        return [];
+    }
+    try {
+        const logs = await publicClient.getLogs({
+            address: contractFactoryAddress,
+            event: parseAbiItem('event BootcampCreated(address indexed bootcampAddress)'),
+            fromBlock: 14619610n,
```

Comment:
can we store the fromBlock as env var instead of hardcoding it? that way, when we redeploy, we wont have to hunt this down in the code to update

Reply:
oh yeah sure, I forgot to do that

Reply:
fixed